xcdat/spatial.py

```diff
@@ -73,10 +73,12 @@ def average(
         keep_weights: bool = False,
         lat_bounds: Optional[RegionAxisBounds] = None,
         lon_bounds: Optional[RegionAxisBounds] = None,
+        lev_bounds: Optional[RegionAxisBounds] = None,
```

Comment:
I think `lev_bounds` is a good generic name, but I'm open to other possibilities.

Reply:
Sounds good to me